tests/common.go

```diff
@@ -5734,3 +5739,61 @@ func SetupTestRail() {
 		log.Debugf("Not all information to connect to testrail is provided, skipping updates to testrail")
 	}
 }
+
+// AsgKillNode terminates the given node
+func AsgKillNode(nodeToKill node.Node) error {
+	initNodes := node.GetStorageDriverNodes()
+	initNodeNames := make([]string, len(initNodes))
+	var err error
+	for _, iNode := range initNodes {
+		initNodeNames = append(initNodeNames, iNode.Name)
+	}
+	stepLog := fmt.Sprintf("Deleting node [%v]", nodeToKill.Name)
+	Step(stepLog, func() {
+		log.InfoD(stepLog)
+		//workaround for eks until EKS sdk is implemented
+		if IsEksCluster() {
+			Inst().N.RunCommandWithNoRetry(nodeToKill, "ifconfig eth0 down  < /dev/null &", node.ConnectionOpts{
+				Timeout:         2 * time.Minute,
+				TimeBeforeRetry: 10 * time.Second,
+			})
+
+		} else {
+			err = Inst().N.DeleteNode(nodeToKill, 5*time.Minute)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Valdiate node %s deletion", nodeToKill.Name))
```

Comment:
Why are we handling error here instead of just returning it?
Is it because it part of the `Step` block?

Reply:
fixed